pages/groups.tsx

```diff
@@ -1,10 +1,15 @@
 import { Identity } from "@semaphore-protocol/core"
 import React, { useCallback, useEffect, useState } from "react"
-import { getMembersGroup, getGroup } from "@/utils/bandadaApi"
+import {
+  getMembersGroup,
+  getGroup,
+  getCredentialGroupJoinUrl
+} from "@/utils/bandadaApi"
 import Stepper from "@/components/stepper"
 import Divider from "@/components/divider"
 import { useSearchParams, useRouter } from "next/navigation"
 import { getRoot } from "@/utils/useSemaphore"
+import { DashboardUrl } from "@bandada/api-sdk"
```

Comment:
This line could be removed and import it from `@/utils/bandadaApi` because this enum can be exported from `@/utils/bandadaApi` like so:

```ts
export { DashboardUrl }
```